### ISSUE TITLE
Anyone get Roland MT-32 Support?

### ISSUE THREAD

**Issue by convolutionbbs** [litchie/dospad#52]
Not an issue as much as a question - there was a post on here from a while back asking for MT-32 support.  Was that ever added?  If not, would it be hard to implement?

Thanks!

**Comment by convolutionbbs** [comment]
Hello,
I'll reply to my original post here.  Is there any way to easily get MT-32 working with dospad?  I have successfully compiled the project in xcode 8 via a target of version 6.  After tweaking the sound section I'm able to get sandblaster support working without any choppiness.

The original post here had a link to this set of files, but there was no instructions or if this ever worked.

I'm not a developer, so any insight would be helpful on what to try.  I've got an iPad Pro so I'm sure it can handle the cpu load. Does the munt source code have to get added in to the dosbox code files and then roms added to function?  Dosbox turbo on android has some flavor of MT-32 emulation working with the addition of the roms.

https://github.com/munt/munt/tree/master/DOSBox-mt32-patch

Thanks.
